MonoGame.Framework/Graphics/RenderTarget2D.cs

```diff
@@ -46,56 +66,24 @@ public RenderTarget2D(GraphicsDevice graphicsDevice, int width, int height)
 			: this(graphicsDevice, width, height, false, SurfaceFormat.Color, DepthFormat.None, 0, RenderTargetUsage.DiscardContents) 
 		{}
 
-        internal RenderTarget2D(GraphicsDevice graphicsDevice,
-                        int width,
-                        int height,
-                        bool mipMap,
-                        SurfaceFormat format,
-                        DepthFormat depthFormat,
-                        int preferredMultiSampleCount,
-                        RenderTargetUsage usage,
-                        SurfaceType surfaceType)
-            : this(graphicsDevice, width, height, mipMap, format, depthFormat, preferredMultiSampleCount, usage, surfaceType, false, 1)
-        {
-        }
-
         /// <summary>
         /// Allows child class to specify the surface type, eg: a swap chain.
         /// </summary>        
-        internal RenderTarget2D(GraphicsDevice graphicsDevice,
+        protected RenderTarget2D(GraphicsDevice graphicsDevice,
```

Comment:
This constructor was used by SwapChanRenderTarget. It doesn't call  PlatformConstruct() because SwapChanRenderTarget does it's own initialization.